prime-angular-frontend/src/app/modules/adjudication/shared/components/notification-email-view/notification-email-view.component.ts

```diff
@@ -58,16 +78,132 @@ export class NotificationEmailViewComponent implements OnInit {
         )
         .subscribe((emailTemplate: EmailTemplate) => {
           this.emailTemplate = emailTemplate;
-          this.editable = false;
+          this.templateEditable = false;
+        });
+    }
+  }
+
+  public saveSubject(): void {
+    if (this.subject.valid) {
+      const data: DialogOptions = {
+        title: 'Save Email Subject',
+        message: `Are you sure you want to overwrite the email subject?`,
+        actionText: 'Save Subject'
+      };
+
+      this.dialog.open(ConfirmDialogComponent, { data })
+        .afterClosed()
+        .pipe(
+          exhaustMap((result: { output: string }) =>
+            (result)
+              ? this.emailTemplateResource.updateEmailSubject(this.route.snapshot.params.eid, this.subject.value)
+              : EMPTY
+          ),
+        )
+        .subscribe((emailTemplate: EmailTemplate) => {
+          this.emailTemplate = emailTemplate;
+          this.subjectEditable = false;
+        });
+    }
+  }
+
+  public saveTitle(): void {
+    if (this.emailTitle.valid) {
+      const data: DialogOptions = {
+        title: 'Save Email Title',
+        message: `Are you sure you want to overwrite the email title?`,
+        actionText: 'Save Title'
```

Comment:
For consistency, I think these should be "Email Template Title"

Reply:
It is updated.